### PR TITLE
filter video for swiper on home page (creation component swiper) and …

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**
…responsive page

### CODE REVIEW

**Review by Fostrankill — DISMISSED** [review]

File: frontend/src/components/SwiperHome/SwiperHome.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import YouTube from "react-youtube";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Mousewheel, Keyboard, FreeMode } from "swiper";
+/* eslint-disable import/no-unresolved */
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+
```
Comment (LoicBrassart) [inline frontend/src/components/SwiperHome/SwiperHome.jsx:12]: Re-enable the ESLint rule here

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Plx fix Evil

File: frontend/package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "react-icons": "^4.7.1",
     "react-router-dom": "^6.4.4",
     "react-youtube": "^10.1.0",
-    "swiper": "^8.4.5"
+    "swiper": "^8.4.5",
+    "template-fullstack": "file:.."
```
Comment (cartige) [inline frontend/package.json:21]: ?

**Review by cartige — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**